nuxeo-core/nuxeo-core-api/src/main/resources/OSGI-INF/legacy-acp-behavior.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.core.security.acl.properties">
+    <extension target="org.nuxeo.runtime.ConfigurationService" point="configuration">
+        <documentation>
+            Enable Legacy behavior (Nuxeo 6.x) on ACL order of execution
```

Comment:
Please explain what that legacy behavior is about exactly: describe the mechanism in legacy and in non-legacy.